### ISSUE TITLE
Heroku Drywall mongoDB 3.2

### ISSUE THREAD

**Issue by MrLeusk** [jedireza/drywall#403]
There is a scheduled update to MongoDB upgrade. It is now 3.0.12 is the default for new mLab deployments and it will be changing to 3.2. 

Recently I cloned and deployed drywall and I am getting a mongodb connection error (see below).  I am wondering if the issue has to do with the upgrade. 

MongoError: failed to connect to server [localhost:27017] on first connect

**Comment by jedireza** [comment]
Thanks for opening an issue. I haven't done the mLab Mongo 3.2 upgrade for `http://drywall.herokuapp.com/` yet.

> MongoError: failed to connect to server [localhost:27017] on first connect

At first I'm suspicious about that `localhost:27017` host/port. Are you sure you're using the environment variable (in my case `MONGOLAB_URI`)?

**Comment by MrLeusk** [comment]
Yes it runs perfectly when I link to an old mongodb. The issue arises when I create a new mongodb add on.

**Comment by jedireza** [comment]
Drywall has been moved to Aqua. Closing this issue.
